src/System.Runtime.WindowsRuntime/src/System/IO/WindowsRuntimeStreamExtensions.CoreRT.cs

```diff
@@ -0,0 +1,361 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
```

Comment:
This file is different just in a couple of lines from the CoreCLR version - because of the BufferedStream vs. BufferedStreamWrapper divergence. It won't be necessary once you fix the unnecessary divergence.